### PR TITLE
Rename blacklist to blocklist

### PR DESCRIPTION
**Pull request by chao-xian**

### CODE REVIEW

File: spec/javascripts/global-bar-init.spec.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ describe('Global bar initialize', function () {
   })
 
   it('does not set cookie for blacklisted URL', function() {
```
Comment (vanitabarrett) [inline spec/javascripts/global-bar-init.spec.js]: Can we replace it here too?

Comment (alex-ju) [inline spec/javascripts/global-bar-init.spec.js]:
```suggestion
  it('does not set cookie for blocklisted URL', function() {
```

Can we replace it for the survey-related code too? 🙏

Comment (chao-xian) [reply spec/javascripts/global-bar-init.spec.js]: Good spot! Thanks

Comment (alex-ju) [reply spec/javascripts/global-bar-init.spec.js]: Ah, Vanita already spotted it.

Comment (chao-xian) [reply spec/javascripts/global-bar-init.spec.js]: Just spotted that - will do

**Pull request merged**